.gitlab-ci.yml

```diff
@@ -1,3 +1,5 @@
+include: '.DOMjudge-release.yml'
```

Comment:
I'd just call it `.release.yml` since the `DOMjudge` part is clear and it would also remove capitalization from the filename, which we don't use anywhere else.

Reply:
fixed,